indico/modules/events/papers/operations.py

```diff
@@ -57,3 +61,21 @@ def update_competences(user_competences, competences):
     event.log(EventLogRealm.management, EventLogKind.positive, 'Papers',
               "Updated competences for user {}".format(user_competences.user.full_name), session.user,
               data={'Competences': ', '.join(competences)})
+
+
+def paper_change_state(contribution, contrib_data, judgment, judge, send_notifications=False):
+    log_data = {'New state': orig_string(judgment.title)}
+    if judgment == PaperAction.accept:
+        contribution.paper_last_revision.state = PaperRevisionState.accepted
+    elif judgment == PaperAction.reject:
+        contribution.paper_last_revision.state = PaperRevisionState.rejected
+    elif judgment == PaperAction.to_be_corrected:
+        contribution.paper_last_revision.state = PaperRevisionState.to_be_corrected
+    PaperReviewComment(paper_revision=contribution.paper_last_revision, user=judge,
```

Comment:
I also thought we settled for adding the judgment comment as a field in the revision. Not that I disagree with this approach, but just wondering if I missed something.